street_lights/index.html

```diff
@@ -2,11 +2,48 @@
 <html>
   <head>
     <meta charset="utf-8">
-    <title>Światła drogowe</title>
+    <title>Swiatła drogowe</title>
   </head>
+<style>
+#streetLights {
```

Comment:
Jak bedziesz przenosił to sformatuj kod w ten sposób:

```css
.klasa {
    wlasciowosc: ok;
}

.nastepna {
    wlasciowosc: super;
}
```

Chodzi o wcięcia i o odstępy miedzy klasami